vmm/src/migration.rs

```diff
@@ -49,10 +49,12 @@ pub fn recv_vm_config(source_url: &str) -> std::result::Result<VmConfig, Migrata
     vm_config_path.push(SNAPSHOT_CONFIG_FILE);
 
     // Try opening the snapshot file
-    let vm_config_file =
+    let mut vm_config_file =
         File::open(vm_config_path).map_err(|e| MigratableError::MigrateSend(e.into()))?;
```

Comment:
@rbradford The error being returned (e.g. `MigrateSend`) looks wrong to me. Shouldn't it be `MigrateReceive` (as it is for `vm_restore()`)?

Reply:
Yeh, that looks like a copy and paste error - i'm happy for @up2wing to address it in this commit / PR

Reply:
Thanks for pointing that, I'm glad to address it.